### PR TITLE
Need to customize new templates

### PR DESCRIPTION
**Pull request by nathanbegbie**
search_results.pt, search_box.pt, paginator.pt and 404.pt

### CODE REVIEW

**Comment by nathanbegbie** [comment]
malaria_no_more___universal_core

malaria_no_more___universal_core

@SteveBarnett Please review. Have made no changes to CSS on this one.

File: unicorecmsmnm/templates/search_box.pt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div xmlns:metal="http://xml.zope.org/namespaces/metal" 
+	 metal:define-macro="search_box"
+	 xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+     i18n:domain="unicorecmsmnm">	
+	<div class="search-box">
+		<form action='/search/' method='GET' class='form-search'>
+		    <input id="q" name='q' type="text" hint='enter search here..' i18n:attributes="hint">
```
Comment (SteveBarnett) [inline unicorecmsmnm/templates/search_box.pt]: Hint can go. :hocho:

File: unicorecmsmnm/templates/search_results.pt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<div metal:use-macro="view.global_template"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+    i18n:domain="unicorecmsmnm">
+    <div metal:fill-slot="content" id="content">
+        <div class="home-content items">
+            <div class="box">
+            <div tal:condition="paginator" class="h2">
+                <p tal:condition="paginator.total_count()==1"><span tal:omit-tag="" i18n:translate='1 search result for:'>1 search result for: </span><b>${query}</b></p>
```
Comment (SteveBarnett) [inline unicorecmsmnm/templates/search_results.pt]: Omit the `<p>` and the `<b>` tags here: neither are needed.

**Comment by SteveBarnett** [comment]
Teeny, tiny, comment, otherwise :+1:

**Comment by nathanbegbie** [comment]
@miltontony  review please

**Comment by nathanbegbie** [comment]
@miltontony @smn pls review

**Comment by miltontony** [comment]
:+1:

**Pull request merged**